shanoir-ng-front/src/app/shared/mass-download/mass-download.service.ts

```diff
@@ -526,6 +526,7 @@ export class MassDownloadService {
         task.progress = 0;
         task.status = 2;
         task.eventType = 'downloadDataset.event';
+        task.sessionId = this.sessionService.sessionId;
```

Comment:
add the id of the current browser tab, so if it is closed we know the task is in error state